pihole

```diff
@@ -234,8 +234,8 @@ Time:
     fi
     echo -e "  ${INFO} Enabling blocking"
     local str="Pi-hole Enabled"
-
-    sed -i "s/^BLOCKING_ENABLED=false/BLOCKING_ENABLED=true/" "${setupVars}"
+    
```

Comment:
Obsolete spaces. Nice `sed` magic btw 👍🙂!

Reply:
All credit to the user on Superuser - I barely understand it :) 

I was going to try and functionise it, as it's used more than once, but... I don't care enough 😂